platform/standalone.go

```diff
@@ -44,7 +44,7 @@ func (s *StandaloneMessageBus) Connect(inst flux.InstanceID) (Platform, error) {
 // trying to use it is the only way to tell if it's closed -- the
 // error representing the cause will be sent to the channel supplied.
 func (s *StandaloneMessageBus) Subscribe(inst flux.InstanceID, p Platform, complete chan<- error) {
-	s.Lock()
+	s.RWMutex.Lock()
```

Comment:
Ah ha. I had plumbed the `(Un)Lock` methods through, which meant I had to rename this. but not anymore!